llvm/test/Transforms/LICM/PR116813-memoryssa-outdated.ll

```diff
@@ -0,0 +1,24 @@
+; RUN: opt -passes='loop-mssa(simple-loop-unswitch<nontrivial>),print<memoryssa>' -verify-memoryssa -disable-output -S < %s 2>&1 | FileCheck %s
```

Comment:
Can you please change this to a normal test using update_test_checks.py? I don't think your check for MemoryDef really adds anything here, as it would *also* be correct to not have the MemoryDef, so this is kind of testing an implementation detail.